processor/src/operations/io_ops.rs

```diff
@@ -91,6 +94,8 @@ impl Process {
         }
         self.stack.shift_left(5);
 
+        self.decoder.set_user_op_helpers(Operation::MLoad, &word);
```

Comment:
The operation here should probably be `MStoreW`. Also, for this operation we should set the helper to the old value at the memory address (not the new value).